notebooks/ceo_area_analysis.ipynb

```diff
@@ -0,0 +1,594 @@
+{
```

Comment:
<p>This remind me of P vs NP :D, I'd personally prefer the state change spelled out e.g. "Not planted, planted" </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nasaharvest/crop-mask/pull/276/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>